src/utils/components/Consoles/components/CloudInitCredentials/InlineCodeClipboardCopy.tsx

```diff
@@ -3,23 +3,33 @@ import React, { FC } from 'react';
 import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
 import { ClipboardCopy } from '@patternfly/react-core';
 
-import { clipboardCopyFunc } from '../../utils/utils';
+import { LINE_FEED } from '../vnc-console/utils/util';
 
-const InlineCodeClipboardCopy: FC<{ clipboardText: string }> = ({ clipboardText }) => {
+type InlineCodeClipboardCopyProps = {
+  clipboardText: string;
+  isCredentialsVisible?: boolean;
+};
+
+const InlineCodeClipboardCopy: FC<InlineCodeClipboardCopyProps> = ({
+  clipboardText,
+  isCredentialsVisible = false,
+}) => {
   const { t } = useKubevirtTranslation();
 
+  const handleCopy = () => {
+    navigator.clipboard.writeText(clipboardText.concat(String.fromCharCode(LINE_FEED)));
```

Comment:
1. please take a look at error handling in the corresponding  read method https://github.com/kubevirt-ui/kubevirt-plugin/blob/15c10986fa04261be46b8e7b0468ee82e16afa8a/src/utils/components/Consoles/utils/utils.ts#L20
2. the method is small but just to keep read and write co-located I would move it to the same utility file